src/services/contracts/safeContracts.ts

```diff
@@ -170,3 +172,16 @@ export const getFallbackHandlerContractInstance = (chainId: string): Compatibili
 
   return new Contract(contractAddress, new Interface(fallbackHandlerDeployment.abi)) as Compatibility_fallback_handler
 }
+
+// Sign messages deployment
+// TODO: Should this be implemented in Core SDK?
```

Comment:
Good idea to add it the core?

Reply:
cc @germartinez

Reply:
There is a refactor that I want to do regarding how the contract instances are returned by the Core SDK and I think it makes sense to add the SignMessageLib there as well

Reply:
Great!!. For now we leave it like this. Make us know when you will be on it

Thanks!!

Reply:
Maybe it'd help putting it in an issue? I've just moved over the other Core SDK requests into issues [here](https://github.com/safe-global/web-core/pull/565#issuecomment-1250728045).